frame/democracy/src/lib.rs

```diff
@@ -729,6 +947,10 @@ decl_module! {
 		///
 		/// NOTE: Used to be called `remove_proxy`.
 		///
+		/// The dispatch origin of this call must be _Signed_.
+		///
+		/// - `proxy`: The account which will be deactivated as proxy.
```

Comment:
I will die on this hill.
```suggestion
		/// - `proxy`: The account that will be deactivated as proxy.
```